### PR TITLE
fix: recent searches cxfeature flag 

### PR DESCRIPTION
**Pull request by lasoh**

### CODE REVIEW

File: projects/storefrontlib/cms-components/navigation/search-box/search-box.component.html [hunk 1]
```diff
@@ -58,100 +58,100 @@
...
-              searchSuggestions: result.suggestions ?? []
-            });
-            updateChosenWord(suggestion)
-          "
-        >
-        </a>
-      </li>
-    </ul>
-  </ng-container>
+  <div class="message" *ngIf="result?.suggestions?.length">
```
Comment (rmch91) [inline projects/storefrontlib/cms-components/navigation/search-box/search-box.component.html]: New element - `<ng-container *cxFeature="searchBoxFeatures.RECENT_SEARCHES_FEATURE">`

File: projects/storefrontlib/cms-components/navigation/search-box/search-box.component.html [hunk 2]
```diff
@@ -58,100 +58,100 @@
...
+      [cxOutletContext]="{
+        search: searchInput.value,
+        searchBoxActive: searchBoxActive,
+        maxRecentSearches: config.maxRecentSearches
+      }"
+    ></ng-template>
   </ng-container>
 
+  <!--RESULT  PRODUCTS-->
+  <div class="message" *ngIf="result.products?.length">
```
Comment (rmch91) [inline projects/storefrontlib/cms-components/navigation/search-box/search-box.component.html]: New element - `<ng-container *cxFeature="searchBoxFeatures.RECENT_SEARCHES_FEATURE">`

**Review by rmch91 — CHANGES_REQUESTED** [review]

File: projects/storefrontlib/cms-components/navigation/search-box/search-box.component.html (hunk 1 above)
Comment (lasoh) [reply projects/storefrontlib/cms-components/navigation/search-box/search-box.component.html]: added `cxFeature` 👍

File: projects/storefrontlib/cms-components/navigation/search-box/search-box.component.html (hunk 2 above)
Comment (lasoh) [reply projects/storefrontlib/cms-components/navigation/search-box/search-box.component.html]: added `cxFeature` 👍

**Review by rmch91 — APPROVED** [review]

**Pull request merged**